src/include/runtime_config.h

```diff
@@ -35,11 +35,12 @@
 #define __DEFAULT_FAIL_MESSAGE__ CFG_STR(12)
 #define __GLOBAL_INCLUDE_FILE__ CFG_STR(13)
 #define __MUD_IP__ CFG_STR(14)
+#define __PORT_CONFIG__ CFG_STR(15)
 
 /*
  * These config settings return an int (ie number)
  */
-#define BASE_CONFIG_INT (BASE_CONFIG_STR + 15)
+#define BASE_CONFIG_INT (BASE_CONFIG_STR + 100)
```

Comment:
@thefallentree
thefallentree 18 seconds ago Member
by changing the number here from 15 to 100 , you effectively changed the argument that people use set_config() and get_config() . Thus this is a compatibility issue.

I think a better way to implment this is to actually implement double ordering. separating user visible number with underlying storage index